Projects/MoveFlow/Sources/Service/OctopusImplementation/MoveFlowClientOctopus.swift

```diff
@@ -169,6 +169,16 @@ extension MovingFlowModel {
 
         self.extraBuildingTypes = data.extraBuildingTypes.compactMap({ $0.rawValue })
 
+        if Dependencies.featureFlags().isAddonsEnabled {
+            /* TODO: REPLACE WITH BE DATA */
+            self.addonInfo = .init(
```

Comment:
Can we add this to localize? We will need it either we use it from clients or BE.